language/move-lang/src/parser/byte_string.rs

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::errors::*;
+use byteorder::{BigEndian, ByteOrder};
+use std::convert::TryFrom;
+
+use crate::parser::syntax::make_loc;
+
+/// Escape sequence.
+enum Sequence {
+    /// \x00
+    Hex(String),
+    ///u{7FFF}
+    Unicode(String),
+    /// \
+    Linebreak,
```

Comment:
Multiline string literals are another feature that I thought might be best left to a follow-up change. This version looks similar to how Rust handles it. I don't have strong opinions here but I think we should have some discussion to make sure we want to adopt this feature. @tnowacki WDYT? If we want to consider this now, maybe we should poll some people who might have opinions and aren't necessarily looking at this PR....

Reply:
Removed support for Multiline string literals.